### DISCUSSION TITLE
Container GUI access with remapped host port through GluetunVPN

### DISCUSSION THREAD

**Discussion by monarc83** [qdm12/gluetun#985] [Q&A]
Hi all, 

I am using GluetunVPN to route the traffic of some containers through a VPN. I am using the "--net=container:GluetunVPN"  parameter and added the ports to the GluetunVPN container so I still can access all of the GUIs. 

Now the problem: I'm trying to setup another container where I have to remap the container GUI port from 8080 to 8090 (host port), but I am unable to access the GUI afterwards. My guess is that it's because I have to change the GUI port, so adding 8090 to the GluetunVPN container does not work.

My Goal is, that the container is using the GluetunVPN VPN and its GUI is still accessible. 

Any advice how to achieve this?

Thanks in advance and BR

**Comment by qdm12** [comment]
Just use `8090:8080` on Gluetun?

If you have changed the internal port of that GUI to i.e. `8090`, then you need to change it on Gluetun too with `8090:8090`.
